lib/auth/auth.go

```diff
@@ -5269,9 +5271,9 @@ func (a *Server) Ping(ctx context.Context) (proto.PingResponse, error) {
 	}
 	features := modules.GetModules().Features().ToProto()
 
-	if a.accessMonitoringEnabled {
-		features.IdentityGovernance = a.accessMonitoringEnabled
-	}
+	// DELETE IN 16.0 and the [func setAccessMonitoringFeatureForOlderClients]
```

Comment:
If we need to perform some cleanup operation in future, I'd consider placing an issue for it (or adding as a task to existing cleanup issue, if such one exists).

Reply:
i don't think we have an issue for it 😅, we just mark them `DELETE IN XXX` and get to them when we can